From ff5cf224850ca1923d1239bdf689afc9f0c3d2c9 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Mon, 26 Sep 2011 11:56:15 +0300
Subject: [PATCH] example.jwmrc: Use clean colors
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit


Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 example.jwmrc |    3 +++
 1 file changed, 3 insertions(+)

--- a/example.jwmrc
+++ b/example.jwmrc
@@ -96,7 +96,10 @@
 
       <Active>
          <Text>white</Text>
+<!-- #DEBIAN
          <Title>#70849d:#2e3a67</Title>
+-->
+         <Title>gray30:gray60</Title>
          <Outline>black</Outline>
          <Opacity>1.0</Opacity>
       </Active>
